install.sh

```diff
@@ -65,8 +65,8 @@ function install() {
     cp "./fanctrl.py" "/usr/local/bin/fw-fanctrl"
     chmod +x "/usr/local/bin/fw-fanctrl"
     chown "$LOGNAME:$LOGNAME" "/usr/local/bin/fw-fanctrl"
-    mkdir -p "$HOME/.config/fw-fanctrl"
-    cp -n "./config.json" "$HOME/.config/fw-fanctrl/" || true
+    mkdir -p "/etc/fw-fanctrl"
+    cp -n "./config.json" "/etc/fw-fanctrl/config.json" || true
```

Comment:
Uninstall needs to be adapted too.

Reply:
I am working on another PR that does includes this change and transition from previous config to /etc.
I will submit it after PR #29 is merged :+1: